src/styles/normalize.scss

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  color: $main-text-color;
+  margin: 0;
+
+  &:hover {
+    transition: 300ms;
```

Comment:
if you'd like to make your transition smooth you should add transition: 300ms; not to hover effect but to static selector
Also it's not a good practice to not specify what exactly you'd like to change

Example:
```
.title {
  color: #ff0000;
  transition: color 300ms;
  
  &:hover {
    color: #00ff00;
  }
}
```